frontend/packages/erd-core/src/features/erd/components/ERDContent/components/TableNode/TableDetail/CollapsibleHeader/CollapsibleHeader.module.css

```diff
@@ -51,6 +51,6 @@
   transition: max-height 0.1s var(--default-timing-function);
 
   /* enable to scroll to the top children element, */
-  padding-top: 40px;
-  margin-top: -40px;
+  padding-top: 41px;
+  margin-top: -41px;
```

Comment:
It looks 41px is right height of the header.
This change make the focus ring be displayed well.

https://github.com/liam-hq/liam/blob/64017c3ebec3156556d5e14fd219bf29f4068fb6/frontend/packages/erd-core/src/features/erd/components/ERDContent/components/TableNode/TableDetail/Indexes/Indexes.tsx#L20-L22

without this change:
<img width="409" height="223" alt="Image" src="https://github.com/user-attachments/assets/0d103dfb-f470-4fab-b8b8-8be96048e0b8" />